<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QWidget" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Normal Settings</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>19</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="static_set_dailyGoals">
         <property name="text">
          <string>Daily Goals</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_set_dailyGoals"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="static_set_speedUpdate">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>ISK Speed Update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="LE_set_speedUpdate">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>Every 3mins</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Every 5mins</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Every 10mins</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Every 15mins</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Every 20mins</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Every 30mins</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Every hour</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Graphic</string>
   </property>
   <widget class="QCheckBox" name="CB_set_showAD">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Do not show corp AD on startup</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CB_set_showProcess">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hide Process Bar</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_save</sender>
   <signal>clicked()</signal>
   <receiver>settings</receiver>
   <slot>set_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>settings</receiver>
   <slot>set_cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>set_save()</slot>
  <slot>set_cancel()</slot>
 </slots>
</ui>
